modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -172,6 +172,11 @@ export default {
         next.data._id = this.value.data._id;
         fieldState._id = { data: this.value.data._id };
       }
+      // Though not *always* in the schema, keep track of the archived status.
```

Comment:
Only not in the schema for images, maybe?

Reply:
Only not in the schema for pages (because with pages, it's a place too, so not in any sense adequately represented by a schema field).